docs/getting_started.md

```diff
@@ -52,13 +52,13 @@ cargo run
 Note: You may also pass environment variable values in the same command as the run command. This is especially useful for setting log levels.
 
 ```sh
-RUST_LOG=debug cargo run -p trin
+RUST_LOG=debug cargo run 
 ```
 
 View CLI options:
 
 ```sh
-cargo run -p trin -- --help
+cargo run -- -h
```

Comment:
I often prefer the long-format flags in documentation for readability. Although help is so universal that this one doesn't really bother me. 👍🏻 to whatever you like.